6.RNAseq/scripts/4.drug_correlation.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+import sys
+
+sys.path.append("../6.RNAseq/utils")
+from analyze_utils import find_drugs
+
+sys.path.append("../5.drug-dependency")
+from utils import load_utils
+
+
+# In[2]:
+
+
+# Define the location of the saved models and output directory for results
+output_dir = pathlib.Path("results")
+output_dir.mkdir(parents=True, exist_ok=True)
+
+# File to store the combined correlation results
+final_test_results_file = output_dir / "test_r2.parquet"
+final_test_predictions_file = output_dir /  "test_preds.parquet"
+
+
+# In[3]:
+
+
+final_test_results_df = pd.read_parquet(final_test_results_file)
+final_test_predictions_df = pd.read_parquet(final_test_predictions_file)
+
+
+# In[4]:
+
+
+meta_cols = ['model', 'latent_dimension', 'z_dimension', 'type', 'init', 'seed']
+
+# Get all other columns, sorted alphabetically
+data_cols = sorted([col for col in final_test_predictions_df.columns if col not in meta_cols])
+
+# Reorder the dataframe
+final_test_predictions_df = final_test_predictions_df[meta_cols + data_cols]
+
+final_test_predictions_df.head()
+
+
+# In[5]:
+
+
+# Load pre-split RNASeq data
+train = pd.read_parquet("./data/RNASeq_train_zscored.parquet")
+val = pd.read_parquet("./data/RNASeq_val_zscored.parquet")
+test = pd.read_parquet("./data/RNASeq_test_zscored.parquet")
+
+
+# In[7]:
+
+
+prism_dir = pathlib.Path("../5.drug-dependency/results")
+prism_dir.mkdir(parents=True, exist_ok=True)
+final_output_file = prism_dir / "combined_latent_drug_correlations.parquet"
```

Comment:
```suggestion
final_output_file = (prism_dir / "combined_latent_drug_correlations.parquet").resolve(strict=True)
```
I am assuming that this file is expected in order for the notebook to work properly.

Reply:
Yes, fixed